### PR TITLE
Add private-database package

### PR DESCRIPTION
**Pull request by frenautvh**
- i'll squash commits after code review and QC

### CODE REVIEW

File: packages/manager/modules/private-database/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```sh
+yarn add @ovh-ux/manager-private-database
+```
+
+## Usage
+
+```js
+import 'angular';
+import privateDatabase from '@ovh-ux/manager-private-database
```
Comment (cbourgois) [inline packages/manager/modules/private-database/README.md]:
```suggestion
import privateDatabase from '@ovh-ux/manager-private-database';
```

File: packages/manager/modules/private-database/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^1.1.0",
+    "@ovh-ux/ovh-utils-angular": "^14.0.0",
+    "@ovh-ux/web-universe-components": "^1.1.0",
+    "@uirouter/angularjs": "^1.0.15",
+    "angular": "1.7.5",
+    "angularjs-scroll-glue": "^2.2.0",
+    "bootstrap": "^3.3.6",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "chart.js": "^2.7.2",
```
Comment (cbourgois) [inline packages/manager/modules/private-database/package.json]: Seems to be a dependency ?

File: packages/manager/modules/private-database/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "font-awesome": "4.7.0",
+    "oclazyload": "^1.1.0",
```
Comment (cbourgois) [inline packages/manager/modules/private-database/package.json]: Is `oclazyload` used ?

File: packages/manager/modules/sms/package.json [hunk 4]
```diff
@@ -38,7 +38,7 @@
     "bootstrap4": "twbs/bootstrap#v4.0.0",
     "font-awesome": "4.7.0",
     "ng-at-internet": "^3.1.1",
-    "ovh-angular-pagination-front": "^6.0.0",
+    "ovh-angular-pagination-front": "^7.0.0",
```
Comment (antleblanc) [inline packages/manager/modules/sms/package.json]: It shouldn't be a part of this pull request I guess.

**Comment by ovh-cds** [comment]
CDS Report Installation#461.0 ■   
* Stage 1
  * Install ■

**Comment by ovh-cds** [comment]
CDS Report Installation#461.1 ■   
* Stage 1
  * Install ■

**Comment by ovh-cds** [comment]
CDS Report installation#462.0 ■   
* Stage 1
  * Install ■

**Comment by ovh-cds** [comment]
CDS Report installation#500.1 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#500.2 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#500.4 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#577.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#626.0 ✘  
* Stage 1
  * Install ✘

**Comment by jleveugle** [comment]
Hello @frenautvh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @frenautvh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @frenautvh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**